src/tutorial-module/interactive-training/hooks/useBindEvents.ts

```diff
@@ -81,37 +81,37 @@ function setupEventListeners(props: SetupEventListenersProps) {
 
     const getPageIdsForElement = (
         element: Element,
-        eventType: EventListenerType
-    ): Maybe<{ pageIds: string[]; trigger?: EventListenerType }> => {
-        const trainingId = element.getAttribute("data-training-id");
-        if (!trainingId) return undefined;
+        eventType: EventType
+    ): { targetIds: string[]; trigger?: EventType } => {
+        const trainingId = element.getAttribute("data-training-id") || "";
```

Comment:
`data-training-id` is hard-coded 5 times along the code. As I guess they all refer the same, we should have a constant and import everywhere.